Dockerfile

```diff
@@ -8,11 +8,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         python3 \
         runit \
         socat \
-    && rm -rf /var/lib/apt/lists/*
+        gpg \
```

Comment:
Shouldn't these be part of the build deps? I don't think they're required at runtime.